app/src/androidTest/java/org/simple/clinic/patient/PatientRepositoryAndroidTest.kt

```diff
@@ -488,19 +491,49 @@ class PatientRepositoryAndroidTest {
 
   @Test
   fun when_patient_is_marked_dead_they_should_not_show_in_search_results() {
-    val patient =
-        patientRepository.saveOngoingEntry(testData.ongoingPatientEntry(fullName = "Ashok Kumar", age = "20"))
-            .andThen(patientRepository.saveOngoingEntryAsPatient())
-            .blockingGet()
+    val patient = patientRepository
+        .saveOngoingEntry(testData.ongoingPatientEntry(fullName = "Ashok Kumar"))
+        .andThen(patientRepository.saveOngoingEntryAsPatient())
+        .blockingGet()
+
+    val searchResults = patientRepository.search(name = "Ashok").blockingFirst()
+    assertThat(searchResults).isNotEmpty()
+    assertThat(searchResults.first().fullName).isEqualTo("Ashok Kumar")
 
     patientRepository.updatePatientStatusToDead(patient.uuid).blockingAwait()
 
-    val searchResult = patientRepository.search(name = "Ashok").blockingFirst()
-    val patientFirst = patientRepository.patient(patient.uuid).blockingFirst()
+    val searchResultsAfterUpdate = patientRepository.search(name = "Ashok").blockingFirst()
+    assertThat(patientRepository.recordCount().blockingFirst()).isEqualTo(1)
+    assertThat(searchResultsAfterUpdate).isEmpty()
+
+    val deadPatient: Patient = patientRepository.patient(patient.uuid)
+        .unwrapJust()
+        .blockingFirst()
 
     assertThat(patientRepository.recordCount().blockingFirst()).isEqualTo(1)
-    assertThat(patientFirst).isNotNull()
-    assertThat(searchResult).isEmpty()
+    assertThat(deadPatient.status).isEqualTo(PatientStatus.DEAD)
+  }
+
+  @Test
+  fun when_patient_is_marked_dead_they_should_marked_as_pending_sync() {
+    val timeOfCreation = Instant.now(testClock)
+
+    val patient = patientRepository
+        .saveOngoingEntry(testData.ongoingPatientEntry(fullName = "Ashok Kumar"))
```

Comment:
We already have a `TestData#patientProfile` which we should be using instead of creating it via an `OngoingPatientEntry`. Should we use that instead?

Reply:
I considered doing that but it only saves 1 line:

```kotlin
val patientProfile = testData.patientProfile(patientName = "Ashok Kumar")
val patient = patientProfile.patient
patientRepository.save(listOf(patientProfile)).blockingAwait()
```